runtime/str.go

```diff
@@ -816,21 +816,26 @@ func strInterpolate(f *Frame, format string, values *Tuple) (*Object, *BaseExcep
 		if matches == nil {
 			return nil, f.RaiseType(ValueErrorType, "invalid format spec")
 		}
-		if matches[7] != "%" && valueIndex >= len(values.elems) {
+		flags, fieldWidth, fieldType := matches[1], matches[2], matches[7]
+		if fieldType != "%" && valueIndex >= len(values.elems) {
 			return nil, f.RaiseType(TypeErrorType, "not enough arguments for format string")
 		}
-		if matches[1] != "" {
-			return nil, f.RaiseType(NotImplementedErrorType, "conversion flags not yet supported")
-		}
-		if matches[2] != "" || matches[4] != "" {
-			return nil, f.RaiseType(NotImplementedErrorType, "field width not yet supported")
+		if (flags != "" && flags != "0") || fieldWidth == "*" || (fieldType != "d" && fieldType != "x" && fieldType != "X") {
+			if flags != "" {
+				fmt.Println("matches1", flags, matches)
```

Comment:
Oops, missed that. I have been a bit busy with other task in company. will remove.